spec/lib/acronyms_spec.rb

```diff
@@ -1,11 +1,11 @@
 require "rails_helper"
-require "abbreviations"
+require "acronyms"
 
-describe Abbreviations, type: :helper do
+describe Acronyms, type: :helper do
   let(:content) { "All prices include VAT except where marked exVAT" }
   let(:abbreviations) { { "VAT" => "Value added tax" } }
```

Comment:
`acronyms`

Reply:
Something odds gone on here - I'd fixed these, must've mucked up the rebase